engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details-info.controller.js

```diff
@@ -131,7 +131,7 @@ angular.module('Bastion.content-hosts').controller('ContentHostDetailsInfoContro
         };
 
         $scope.convertMemToGB = function (memoryValue) {
-            return (memoryValue / 1024000).toFixed(2);
+            return (memoryValue/1000000).toFixed(2);
```

Comment:
@thomasmckay thanks for reviewing it. My intention is to convert the reported memory into decimal format. see [1]

 [1] - https://www.gbmb.org/kb-to-gb